dev/com.ibm.ws.jdbc_fat_krb5/publish/servers/com.ibm.ws.jdbc.fat.krb5/server.xml

```diff
@@ -0,0 +1,65 @@
+<server>
+  <featureManager>
+    <feature>componenttest-1.0</feature>
+    <feature>servlet-4.0</feature>
+    <feature>jdbc-4.2</feature>
+    <feature>appSecurity-2.0</feature>
+    <feature>jca-1.7</feature>
+    <feature>cdi-2.0</feature>
+  </featureManager>
+    
+  <include location="../fatTestPorts.xml"/>
+
+  <webApplication location="db2fat.war">
+    <application-bnd>
+      <security-role name="Manager" id="Manager">
+        <user name="user1"/>
+      </security-role>
+      <security-role name="Employee" id="Employee">
+        <user name="user2"/>
+      </security-role>
+    </application-bnd>
+  </webApplication>
+    
+  <library id="DB2JCCLib">
+    <fileset dir="${server.config.dir}/db2"/>
+  </library>
+  
+  <!-- Dummy lib to satisfy login module config. This will be removed eventually -->
+  <library id="loginLib">
+  </library>
+  
+  <basicRegistry realm="defaultRealm">
+    <user password="password" name="user1"/>
+    <user password="password" name="user2" />
+  </basicRegistry>
+  
+  <!-- Must be named JaasClient for the JCC driver -->
+  <jaasLoginContextEntry id="JaasClient" name="JaasClient" loginModuleRef="krb5LoginModule" />
+    <!-- OpenJ9/Hotspot loginModule config -->
+	<jaasLoginModule id="krb5LoginModule" className="com.sun.security.auth.module.Krb5LoginModule" controlFlag="REQUIRED" libraryRef="loginLib">
+    <options credsType="both" useKeyTab="true" keyTab="${server.config.dir}/security/krb5.keytab" principal="${KRB5_USER}" doNotPrompt="true"/>
+  </jaasLoginModule>
+  
+  <!-- IBM Java loginModule config -->
+  <!-- 
+  <jaasLoginModule id="krb5LoginModule" className="com.ibm.security.auth.module.Krb5LoginModule" controlFlag="REQUIRED" libraryRef="loginLib">
+    <options credsType="both" useKeytab="/etc/krb5.keytab" principal="db2user/websphere@EXAMPLE.COM"/>
```

Comment:
Is this here as an example of how a user would configure their options?  Or is this just left over from testing?

Reply:
This is left as an example that I copied from Mark's repo for now. Which reminds me that this config only works for hotspot/J9, so I need to add a skip for IBM JDK since that will require different config (I'll do a separate PR for that)